CMakeLists.txt

```diff
@@ -91,3 +91,7 @@ macro (install_hook)
 endmacro (install_hook)
 
 include (OpmLibMain)
+
+add_test( NAME valgrind-restart
+          COMMAND valgrind ${PROJECT_BINARY_DIR}/bin/test_Restart
+          WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/tests)
```

Comment:
I guess you need to add `CONDITION ${VALGRIND_FOUND}` here because this will only work if valgrind is installed...